### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by Mantelijo**
Grafana/Prometheus/Cadvisor metrics stack for swarm. Metrics services are isolated from the outside world. Grafana instance can be exposed via localhost tunnel by using `grafana-tunnel` subcommand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,3 +404,22 @@ D8X-CLI is not tested on Windows and will probably not work, we would
...
+
+Metrics stack will be deployed automatically when running `d8x setup` command.
+Or you can manually deploy metrics via `d8x setup metrics-deploy` command.
+
+## Metrics services ports
+
+- Grafana is published at `127.0.0.1` on port `4002` on manager node
+- Cadvisor instances are deployed on port `4003` on each worker node.
+
+By default Prometheus instance is not published and is only accessible from
```
Comment (b1674927) [inline README.md:423]: typo: accessible

File: README.md [hunk 2]
```diff
@@ -387,3 +404,22 @@ D8X-CLI is not tested on Windows and will probably not work, we would
...
+grafana instance. Grafana and cadvisor ports are not accesible to the public
```
Comment (b1674927) [inline README.md:424]: typo: accessible

**Review by b1674927 — APPROVED** [review]
please fix small typos

**Comment by Mantelijo** [comment]
typos fixed

**Pull request merged**